### ISSUE TITLE
Data mapper - provide a list view for all the mapped fields 

### ISSUE THREAD

**Issue by dongniwang** [syndesisio/syndesis-ux#37]

**Comment by gashcrumb** [comment]
Think that's in there already @igarashitm correct me if I'm wrong but it's one of the icons above the detail display.  Maybe it just needs to be more obvious that those icons do things :-)

**Comment by dongniwang** [comment]
@gashcrumb thanks for letting me know! 

Now I see the icon for the list view. Would be great if the list view is available without an extra click and replacing the main mapping area. Need to find a better solution here. 

![image](https://user-images.githubusercontent.com/24943812/29982176-b7861854-8f1e-11e7-8939-5c8f60f596e1.png)

**Comment by gashcrumb** [comment]
Yeah, maybe the list view should be the default with an 'Add Mapping...' button as the empty state + at the bottom of the list when there's entries?  Kind of similar to the basic filter page maybe...
